src/Kiota.Builder/Writers/Php/PhpConventionService.cs

```diff
@@ -96,7 +96,7 @@ public override string GetParameterSignature(CodeParameter parameter, CodeElemen
                 CodeParameterKind.ResponseHandler => $"ResponseHandler {GetParameterName(parameter)}",
                 CodeParameterKind.RequestConfiguration => $"{parameter!.Type.Name.ToFirstCharacterUpperCase()} {GetParameterName(parameter)}",
                 CodeParameterKind.Serializer => $"SerializationWriter {GetParameterName(parameter)}",
-                CodeParameterKind.BackingStore => $"BackingStore {GetParameterName(parameter)}",
+                CodeParameterKind.BackingStore => $"{parameter.Type.Name.ToFirstCharacterUpperCase()} {GetParameterName(parameter)}",
```

Comment:
We could just eliminate this branch entirely and use the `_` branch.
```suggestion
```

Reply:
Implemented in latest commits